frontend/micro-ui/web/micro-ui-internals/packages/modules/core/src/pages/citizen/index.js

```
@@ -96,36 +345,330 @@ const Home = ({
+        padding: "15px 20px",
+        textAlign: "center",
+        marginTop: "40px",
+      },
+    };
+
+
     return (
+      // http://localhost:3000/digit-ui/citizen/pt/property/citizen-search
+      // digit-ui/citizen/pt/property/my-applications
+      // /digit-ui/citizen/pt/property/new-application/info
       <React.Fragment>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)